### PR TITLE
update custom chain information

### PR DESCRIPTION
**Pull request by warengonzaga**
- update `paramaters.ts`
- update the `README.md` file.

### CODE REVIEW

**Comment by warengonzaga** [comment]
Hey @MananTank and @nachoiacovino can you review this? Thanks!

**Comment by warengonzaga** [comment]
Hey @MananTank I updated the info!

File: src/consts/parameters.ts [hunk 1]
```diff
@@ -5,10 +5,19 @@
 export const contractConst = "0x436492DBc2E30E56FaC8F2297BD1964833c0687d";
 
 // The name of the chain your contract is deployed to.
-// Provide a string for standard chains, or import one from @thirdweb-dev/chains package.
-// Learn more: https://portal.thirdweb.com/react/react.thirdwebprovider#activechain-recommended
+// For standard chains. (e.g. "mainnet", "rinkeby", "polygon", "mumbai", "base")
+// For custom chains, provide the chain name from thirdweb chain list or use the import name from @thirdweb-dev/chains.
+// Read: https://portal.thirdweb.com/react/react.thirdwebprovider#activechain-recommended
+// Chainlist: https://thirdweb.com/chains
+// e.g."{Sepolia}" or "{Gnosis}"
```
Comment (MananTank) [inline src/consts/parameters.ts]: Don't need the `{` `}` here

File: README.md [hunk 2]
```diff
@@ -32,6 +32,10 @@ Go to the [`parameters.ts`](/src/consts/parameters.ts) and update the following
 1. `contractConst`: The smart contract address of your NFT drop.
 2. `chainConst`: The name of the chain that your smart contract is deployed to.
 
+> **Note:** If you are using a custom chain, you need to provide the chain name from the [thirdweb chain list](https://thirdweb.com/chains). For example, if you are using the **Sepolia** chain you need to use `{Sepolia}` as the chain name. Which is indicated here: <https://thirdweb.com/base>
```
Comment (MananTank) [inline README.md]: We should link to https://thirdweb.com/sepolia instead

Comment (MananTank) [reply README.md]: and don't need to have the `{` `}` here in `{Sepolia}`

File: src/consts/parameters.ts (hunk 1 above)
Comment (warengonzaga) [reply src/consts/parameters.ts]: It doesn't work without `{``}` if it is custom chain.

Comment (MananTank) [reply src/consts/parameters.ts]: I'm not saying it should be a string - `"Sepolia"`, I'm saying it should be an object - `Sepolia`

Example: 

```ts
export const chainConst = Sepolia;
```

Comment (warengonzaga) [reply src/consts/parameters.ts]: I see, thanks for the correction!

**Pull request merged**